File: kotlin/src/main/kotlin/ModuleA.kt
class ModuleA {
    fun test(): String {
        return "youpii"
    }

    fun sayYes(): String {
        return "No"
    }

    fun openTheDoor(key: String): String {
        return when (key) {
            "trash your mouse" -> "unlock"
            else -> "locked"
        }
    }

    fun theFirstVerseOfTheSongIs(firstVerse: String): Boolean {
        return firstVerse == "Somewhere over the rainbow"
    }
}

File: python/src/module_a.py
class ModuleA:

    def test(self):
        return "youpii"

    def sayYes(self):
        return "no"

    def theFirstVerseOfTheSongIs(self, first_verse):
        return "somewhere over the rainbow" in first_verse.lower()

    def open_the_door(self, key):
        if "trash your mouse" == key:
            return "unlock"
        return "locked"


IMPORT_ME_1 = "hello"
IMPORT_ME_2 = "world"
IMPORT_ME_3 = "!"

IMPORT_ME_BUT_I_HAVE_A_VERY_LONG_NAME = "hello world !"